django_lightweight_queue/utils.py

```diff
@@ -33,6 +34,11 @@
 
 
 def load_extra_config(file_path: str) -> None:
+
+    class Overrides(Settings):
+        """Container for holding internally overridden settings."""
+        pass
+
```

Comment:
It makes mypy not throw a fit. There's no functionality-based reason to do this.

Reply:
Two reasons: it makes mypy not throw a fit and since we search by getattr, starting with an empty class means that we can add whatever settings we want and if the getattr fails then it'll fall through to the next layer.

(There was originally a different comment here but it took me a minute to remember why I did it this way.)

Reply:
We need the `_uses_short_names` key from the base class so that we look up the right things, and I didn't want to duplicate it in case it ever needed to change 🤷